ui/src/tool-panel/location-track/splitting/location-track-split.tsx

```diff
@@ -185,7 +185,6 @@ export const LocationTrackSplit: React.FC<SplitProps> = ({
     const isShortNonOverlappingDuplicateLength =
         isPartialDuplicate &&
         nonOverlappingDuplicateLength != undefined &&
-        nonOverlappingDuplicateLength < nonOverlappingDuplicateLength != undefined &&
```

Comment:
Korjaa `Unexpected constant binary expression. Compares constantly with the right-hand side of the '!=' `

Tässähän tuo rivin vasen puoli vertaa itseään itseensä, joka on aina false. Ajonaikana false != undefined on taas aina true -> rivillä ei ole vaikutusta.